sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -388,7 +394,10 @@ export class BatchingReceiverLite {
         logger.verbose(
           `${loggingPrefix} Closing. Resolving with ${brokeredMessages.length} messages.`
         );
-
+        if (drainTimer) {
+          logger.verbose(`${loggingPrefix} clearing drain timer in close handler.`);
```

Comment:
Is this helpful to the customer? I feel like it is more of a sanity check rather than something the customer may be interested in.

Reply:
It is likely for myself while debugging. Removing it.